### DISCUSSION TITLE
Object property assignment shorthand

### DISCUSSION THREAD

**Discussion by tmtron** [josdejong/mathjs#2889] [Ideas]
It would be nice to have object property assignment shorthand notation like in [javascript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions)

Currently we must write:
```
x = 12
obj = { x: x }
```

Possible shorthand notation:
```
x = 12
obj = { x }
```

**Comment by josdejong** [comment]
Thanks for your suggestion @tmtron , I would love that! I use it heavily myself in JavaScript.

Anyone interested in implementing this? Help would be welcome.
